mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/InstantiationUtil.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.mantisrx.server.worker;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+
+public class InstantiationUtil {
```

Comment:
Did we copy this whole class from somewhere? Please add javadoc on why we need such as class.

Reply:
Done